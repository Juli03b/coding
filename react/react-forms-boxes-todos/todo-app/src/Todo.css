li {
    margin-bottom: .2%;
}
p {
    display: inline-block;
}