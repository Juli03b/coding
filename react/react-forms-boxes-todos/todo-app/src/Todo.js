import "./Todo.css"

const Todo = ({todo, removeTodo, editTodo}) => (
    <li>
        <p onClick={editTodo} data-testid="todo">{todo}</p>
        <button onClick={removeTodo}>X</button>
    </li>
);

export default Todo;