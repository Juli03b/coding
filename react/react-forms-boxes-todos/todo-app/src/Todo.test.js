import { render } from "@testing-library/react";
import Todo from "./Todo";

it("Should render", () => {
    render(<Todo />);    
});

it("Should match snapshots", () => {
    const { asFragment } = render(<Todo />);    

    expect(asFragment()).toMatchSnapshot();
});
