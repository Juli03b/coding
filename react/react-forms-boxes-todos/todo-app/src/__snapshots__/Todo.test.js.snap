// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should match snapshots 1`] = `
<DocumentFragment>
  <li>
    <p
      data-testid="todo"
    />
    <button>
      X
    </button>
  </li>
</DocumentFragment>
`;
